Application/ILookupDataService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Domain;
+using Domain.Search;
+
+namespace Application;
+
+public interface ILookupDataService
```

Comment:
Can we move this interface under the `Common/Interfaces` folder? Thanks!

Reply:
Done